### DISCUSSION TITLE
Solax X3-Hybrid Phase Unbalanced not working when Remotecontrol enabled

### DISCUSSION THREAD

**Discussion by Zdenek45** [wills106/homeassistant-solax-modbus#954] [Q&A]
Due to regulations here in Czech Rep. it is necessary to use asymmetrical mode (Phase unbalanced inverter's setting enabled / integration's corresponding setting Phase Power Balance X3 disabled).  
In Remotecontrol Power Control modes (tested Self use, Feed in priority, Battery control) inverter switches to symmetrical mode and this results in asymmetric measured power. In this mode there are situations when I am billed for import on some phases and paid (much less) for export on other phases. This makes Remotecontrol Power Control useless for me.
Can anyone confirm this is by design or is it changed in newer firmwares?

Device info:
X3-Hybrid-15.0
Firmware: DSP v1.27 ARM v1.26
Hardware: Gen4
Serial number: H34A15

Thanks for any advice.

![image](https://github.com/wills106/homeassistant-solax-modbus/assets/152689077/09cd0062-2cac-4adb-b8ff-3a62a88eb0a3)

**Comment by Xeudoron** [comment]
Hi @Zdenek45 ,

I have limited experience with your specific DSP and ARM versions, but I have great experience with DSP 1.35 and ARM 1.34 on various inverters. I encountered issues with earlier factory versions, as I couldn't control the inverter properly. According to the information here https://homeassistant-solax-modbus.readthedocs.io/en/latest/solax-firmware-update, I recommend upgrading the firmware to those specific versions. Be cautious during the upgrade process to avoid potential problems, but I have successfully upgraded several inverters and am currently testing versions 1.42/1.41 without any issues.

The only problem I faced with symmetric load balancing was due to an incorrect naming issue in the HA Solax plugin, which led to a higher electricity bill for one month. Apart from that, I haven't had any problems with asymmetric loads in any plugin or inverter operation mode.

Don't hesitate to ask if you have any further questions.

**Reply by Zdenek45** [reply]
Thank you for the information that my issue can be corrected with fw update. As my inverter is covered by extended warranty I instructed my supplier to update the inverter remotely by the manufacturer.
I knew my inverters firmware versions are quite old, outside the scope of changelog on the mentioned site. For some time I am no longer a fan of frequent and unnecessary sw updates of any kind. This was the reason I asked the question.
I let others know if my issue is resolved as soon as my inverter is upgraded to new firmware.

**Reply by martinviktora** [reply]
The same problem with Firmware: DSP v1.41 ARM v1.40

**Comment by Zdenek45** [comment]
I manually upgraded to firmware DSP v1.42 ARM v1.41. I can confirm same behavior as before. As 
soon the inverter is in Modbus Power Control = Enabled mode, the inverter switches to symmetrical mode.

**Comment by davidrus** [comment]
Everything is still the same with DSP v1.46 and ARM v1.44.
Could someone confirm if this will be addressed in future versions?

Thanks in advance!

**Comment by petrwhiski** [comment]
I confirm the behavior. I have just upgraded to DSP v1.50 ARM v1.47 and nothing changed. In remotecontrol enabled and "Enabled Feedin Priority" the SolaX Inverter Modbus integration regulates same power on all phases :-(

**Comment by petrwhiski** [comment]
Please see attached. 
### Remotecontrol on
Solax integration puts same power on all 3 phases (first 3 meters), but the load is only on the 3rd phase (second row):
![remotecontrol on](https://github.com/user-attachments/assets/f6652b14-c31a-4cee-bf40-2ac0996f349b)

### Without remotecontrol
Solax correctly utilizes phases so house has (almost) zero result:
![remotecontrol off](https://github.com/user-attachments/assets/985f95dc-a51e-4510-8c25-35bf12158276)

**Comment by petrwhiski** [comment]
Ok, now I know why. Thumbs up for relicot :-)
[https://github.com/wills106/homeassistant-solax-modbus/issues/1272](url)
